wagl/brdf.py

```diff
@@ -415,7 +422,48 @@ def segmentize_polygon(src_poly, length_scale):
     return wkt.loads(src_poly_geom.ExportToWkt())
 
 
-def only_ocean_pixels(src_poly, src_crs, fid_mask):
+def read_copernicus_wbm(cop_pathname, dst_geobox):
+    OCEAN = 1
+    NODATA = 10  # outside the valid range 0-3
+
+    result = np.full(dst_geobox.shape, NODATA, dtype=np.uint8)
+    dst_crs = dst_geobox.crs.ExportToProj4()
+
+    for dataset_reader in list_copernicus_bands_for_geobox(
+        cop_pathname, copernicus_wbm_image_for_latlon, dst_geobox
+    ):
+        with dataset_reader as ds:
+            subset, subset_geobox = covering_geobox_subset(
+                dst_geobox, GriddedGeoBox.from_dataset(ds)
+            )
+
+            if subset is None:
+                continue
+
+            try:
+                reprojected = np.full(subset_geobox.shape, NODATA, dtype=np.uint8)
+
+                reproject(
+                    source=rasterio.band(ds, 1),
+                    destination=reprojected,
+                    dst_crs=dst_crs,
+                    dst_transform=subset_geobox.transform,
+                    dst_nodata=NODATA,
+                    resampling=Resampling.mode,
```

Comment:
`Resampling.mode` seems like a sane choice. Did you do any basic comparisons of `mode` vs `NN`?

Reply:
It was David's recommendation.